doc/languages-frameworks/rust.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Rust
+author: Matthias Beyer
+date: 2017-03-05
+---
+
+# User's Guide to the Rust Infrastructure
+
+Rust is packaged with nixpkgs, although there are no rust crates packaged in
+nixpkgs.
+Therefor, one has to use `cargo` directly.
```

Comment:
While crates are not precompiled there is `buildRustPackage` to build Rust projects:

```
$ ag buildRustPack | wc -l       
27
```